src/applications/vre/28-8832/containers/WizardContainer.jsx

```diff
@@ -28,7 +28,7 @@ const WizardContainer = props => {
           28-8832
```

Comment:
Update here?

Reply:
Keeping the content the same as this app is 28-8832, but fixed spacing.